### PR TITLE
Outstanding OTEL questions

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Unresolved OTEL docs questions - need SME input.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Sumo Logic Distribution for OpenTelemetry stores the configuration for the colle
 | :-- | :-- |
 | Linux | `/etc/otelcol-sumo/conf.d` |
 | Mac | `/etc/otelcol-sumo/conf.d` |
-| Windows | `C:\ProgramData\Sumo Logic OpenTelemetry Collector\` |
+| Windows | `C:\ProgramData\Sumo Logic OpenTelemetry Collector\` |
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:16]: [@sumoanema](https://github.com/sumoanema) [@arunpatyal](https://github.com/arunpatyal) [@ajaiswals](https://github.com/ajaiswals) , can you confirm the Windows path mentioned above is correct? cc @angadrandhawa1

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 2]
```diff
@@ -42,7 +42,9 @@ Any configuration for a Source (e.g., MySQL, Nginx, Application Logs) should be
 It is recommended to maintain the configuration of all reusable components in `conf.d/common.yaml`. For example, a file named `conf.d/mysql.yaml` can contain the MySQL receiver along with any processors that are intended to modify the collected data before sending it to Sumo Logic.
 :::
 
-The **App Catalog** provides a mechanism to create these configuration files using a simple UI form input.
+The **App Catalog** provides a mechanism to create these configuration files using a simple UI form input. Learn More.
+
+<!-- Learn More should be a link to App Catalog page that explains how the Source Configuration section works.-->
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:47]: @angadrandhawa1 can you clarify what you meant in this comment?

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 3]
```diff
@@ -53,6 +55,8 @@ There are some processors provided in `sumologic.yaml` that are intended to be u
 * **Memory limiter processor**. It is used to prevent out-of-memory situations on the collector. It should be always first on the processor's list. For more information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
 * **Batch processor**. It accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. See [Using batch processor to batch data](#using-batch-processor-to-batch-data) for more information.
 
+<!--There is another processor called ResourceDetection/system. Should that also be used?-->
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:58]: @angadrandhawa1 is this still needed?

Comment (angadrandhawa1) [reply docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:58]: Yes we should call it our the ResourceDetection/System and explain what it does. @arunpatyal , @sumoanema can provide more details about the processor and how it should be used.

Comment (sumoanema) [reply docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:58]: The [resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/) can be used to detect resource information from the host,
in a format that conforms to the [OpenTelemetry resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions/), and append or override the resource value in telemetry data with this information. This processor is used to tag labels like :
* host.name
* host.id
* os.type

Comment (kimsauce) [reply docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:58]: @angadrandhawa1 @sumoanema thank you both!

**Comment by jpipkin1** [comment]
@kimsauce, I can review this once you update the PR per SME input.

**Pull request merged**